static/context.html

```diff
@@ -16,13 +16,96 @@
        before body, Angular Scenario test framework fails to find the body and crashes and burns in
        an epic manner. -->
   <script type="text/javascript">
-    // sets window.__karma__ and overrides console and error handling
-    // Use window.opener if this was opened by someone else - in a new window
-    if (window.opener) {
-      window.opener.karma.setupContext(window);
-    } else {
-      window.parent.karma.setupContext(window);
-    }
+    (function () {
```

Comment:
Yea, definitely. This was mostly to get the proof of concept up and out the door =)